app/views/lectures/show.html.erb

```diff
@@ -1,51 +1,54 @@
-<h5><%= @lecture.name %></h5>
-<% if @lecture.start_time &&  @lecture.end_time && @lecture.date%>
-  <p><%= @lecture.date.strftime("%d.%m.%Y") %> <%= @lecture.start_time.strftime("%H:%M -") %> <%= @lecture.end_time.strftime("%H:%M") %></p>
-<% end %>
+<div class="container">
```

Comment:
The whole Dashboard is missing and 16 tests are failing. All failing tests are in the lectures/show.html.erb_spec.rb. I guess they should be fixed if you add the Dashboard

Reply:
![image](https://user-images.githubusercontent.com/39387258/73057176-1b742d00-3e91-11ea-976d-528539cf4d45.png)

Reply:
sorry, that must have happend during the latest merge. Haven't tested it. I will fix this!

Reply:
Yeah the navbar rendering got lost during merge. Should be fixed now.